### DISCUSSION TITLE
@types/deno

### DISCUSSION THREAD

**Discussion by kasperisager** [DefinitelyTyped/DefinitelyTyped#56543] [Request a new @types package]
Originally requested in #45597. I'm currently working on a library with bindings to both web, Node, and Deno. For web and Node, I can pull in `@types/web` and `@types/node`, but for Deno I currently need to manually output Deno's declarations (`deno types`) and point to them via `typeRoots`. Being able to just pull in `@types/deno` would be fantastic.

**Comment by Richienb** [comment]
Unlike most other typings packages, `@types/web` is automatically generated from https://github.com/microsoft/TypeScript-DOM-Lib-Generator. Something similar will need to be done here.

**Comment by RobertAKARobin** [comment]
Filed a related issue here: https://github.com/denoland/deno/issues/17028

**Comment by RobertAKARobin** [comment]
FWIW all the relevant Deno types seem to be [in one file](https://github.com/denoland/deno/blob/main/cli/tsc/dts/lib.deno.ns.d.ts), so perhaps this isn't such a big lift?
